examples/TestDriver/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "devDependencies": {
     "babel-jest": "23.6.0",
-    "detox": "12.1.1",
+    "detox": "18.9.0",
```

Comment:
it looks like heap's repo has detox@18.2.2, should we keep this consistent across RN + other repo?

Reply:
I can upgrade heap's version to the latest.